src/include/desktopentry.h

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: 2023 UnionTech Software Technology Co., Ltd.
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#include <QString>
+#include <QMap>
+#include <QVariant>
+#include <QMetaType>
+#include <QDebug>
+#include <QLocale>
+
+constexpr static auto defaultKeyStr = "default";
```

Comment:
should this be just a empty string?

Reply:
I think the empty string may confuse the user